src/LottieRenderer.ts

```diff
@@ -0,0 +1,327 @@
+import {
+	CompLottieLayer,
+	SpriteLottieLayer,
+	Tools
+} from './core';
+import { MeshBatcher } from './MeshBatcher';
+import { Matrix, Quaternion, Texture2D, Vector3 } from "oasis-engine";
+import { Script } from "oasis-engine";
+import { LottieResource } from './LottieResource';
+
+export { LottieLoader } from './LottieLoader';
+
+export class LottieRenderer extends Script {
+	static unitsPerPixel: number = 1 / 128;
+
+	repeats = 0;
```

Comment:
if is public, you should add comments, if internal should  add "_" and `@internal`, if private should add "_" and `private`.